js/main.js

```diff
@@ -1,13 +1,84 @@
+var $front = document.querySelector('#front-page');
+var $ajaxList = document.querySelector('#ajax-list');
+var $ul = document.querySelector('ul');
+
 function categorySelect(event) {
-  if (event.target.className === 'chars' || event.target.getAttribute('id') === 'chars') {
-    event.target.className = 'chars';
-  } else if (event.target.className === 'locations' || event.target.getAttribute('id') === 'locations') {
-    event.target.className = 'locations';
-  } else if (event.target.className === 'episodes' || event.target.getAttribute('id') === 'episodes') {
-    event.target.className = 'episodes';
-  } else if (event.target.className === 'favs' || event.target.getAttribute('id') === 'favs') {
-    event.target.className = 'favs';
+  if (event.target.getAttribute('id') === 'chars-img' || event.target.getAttribute('id') === 'chars') {
+    $front.className = 'hidden';
+    $ajaxList.className = 'ajax-list';
+    var char = 'character';
+    getAPIData(char);
+  } else if (event.target.getAttribute('id') === 'locations-img' || event.target.getAttribute('id') === 'locations') {
+    $front.className = 'hidden';
+    $ajaxList.className = 'ajax-list';
+    var location = 'location';
+    getAPIData(location);
+  } else if (event.target.getAttribute('id') === 'episodes-img' || event.target.getAttribute('id') === 'episodes') {
+    $front.className = 'hidden';
+    $ajaxList.className = 'ajax-list';
+    var episode = 'episode';
+    getAPIData(episode);
+  } else if (event.target.getAttribute('id') === 'favs-img' || event.target.getAttribute('id') === 'favs') {
+    $front.className = 'hidden';
   }
 }
 
 window.addEventListener('click', categorySelect);
+
+var xhr;
+
+function getAPIData(category) {
+  xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://rickandmortyapi.com/api/' + category);
+  xhr.responseType = 'json';
+
+  xhr.addEventListener('load', function () {
+    loadDOM();
+    window.addEventListener('DOMContentLoaded', loadDOM);
```

Comment:
This line won't be necessary. The data will come back from this request long after the DOM content has been loaded.